skimage/measure/mc_meta/visual_test.py

```diff
@@ -0,0 +1,72 @@
+""" 

```

Comment:
This is too much for a quick docstring example, and too technical (and 3D) for the example gallery. It's a very useful thing for dev-to-dev communication though. We don't have much that falls into this category, so not a lot of precedent. However, as it isn't publicly exposed, I'm OK having it here if @stefanv doesn't veto.